Plugins/BackbonePluginTemplatePlugin/BackbonePluginTemplatePluginGenerator.cs

```diff
@@ -76,6 +89,44 @@ private Task AddProjectsToSolution()
                 new[] { projectInfo, projectTestInfo });
         }
 
+        private Task AddInternalNugetServerToNugetConfig()
+        {
+            var nugetConfigFileName = "NuGet.config";
+            var nugetConfigFilePath = Path.Combine(SolutionDirectory, nugetConfigFileName);
+
+            var nugetConfigModification = CreateInternalNugetServerModification();
+            return _nugetConfigurationManipulator.ModifyNugetConfiguration(nugetConfigFilePath, nugetConfigModification);
+        }
+
+        private NugetConfigurationModification CreateInternalNugetServerModification()
+            => new NugetConfigurationModificationBuilder(_nugetPackageSourceManipulator)
+                .AddPackageSource(CreateInternalNugetServer())
+                .Build();
+
+        private NugetPackageSource CreateInternalNugetServer()
+            => new NugetPackageSource("Internal Nuget Server", "https://nuget.messerli.ch/v3/index.json");
+
+        private Task AddMsBuildSdkToGlobalJson()
+        {
+            var globalJsonFileName = "global.json";
+            var globalJsonFilePath = Path.Combine(SolutionDirectory, globalJsonFileName);
+
+            var globalJsonModification = CreateMsBuildSdk();
+            return _globalJsonManipulator.ModifyGlobalJson(globalJsonFilePath, globalJsonModification);
+        }
+
+        private GlobalJsonModification CreateMsBuildSdk()
+            => new GlobalJsonModificationBuilder()
+                .AddMsBuildSdk(CreateBackbonePluginSdk())
+                .AddMsBuildSdk(CreateCentralPackageVersionSdk())
```

Comment:
Ask the user if they want to use central package versions or not

Reply:
Addressed in: [Ask user if they want use central package versions](https://github.com/messerli-informatik-ag/meta-generator/pull/22/commits/085d48115c5581cd9b6010f787bd2430dfc4d77e)